src/validators/v1/public.js

```
@@ -0,0 +1,13 @@
+/**
+ * name : validators/v1/public.js
+ * Description : Validations of public controller
+ */
+module.exports = {
+	branding: (req) => {},
```

ESLint

[error] 'req' is defined but never used.

(no-unused-vars)